app/scripts/controllers/network/network-controller.js

```diff
@@ -406,12 +409,17 @@ export default class NetworkController extends EventEmitter {
 
   _configureProvider({ type, rpcUrl, chainId }) {
     // infura type-based endpoints
-    const isInfura = INFURA_PROVIDER_TYPES.includes(type);
+    const isInfura =
+      INFURA_PROVIDER_TYPES.includes(type) &&
+      type !== NETWORK_TYPES.CONSENSYS_ZKEVM;
+
     if (isInfura) {
       this._configureInfuraProvider(type, this._infuraProjectId);
       // url-based rpc endpoints
     } else if (type === NETWORK_TYPES.RPC) {
       this._configureStandardProvider(rpcUrl, chainId);
+    } else if (type === NETWORK_TYPES.CONSENSYS_ZKEVM) {
```

Comment:
why doesn't the `isInfura` case work here?

Reply:
oh, actually, we do want to call _configureStandardProvider for this, so we just should not include this network in `INFURA_PROVIDER_TYPES`

Reply:
because the consenSys zkEVM network is not considered as a supported by Infura in this repository https://github.com/MetaMask/eth-json-rpc-infura
I created a PR in https://github.com/MetaMask/eth-json-rpc-infura repository to fix this issue. Once this PR (https://github.com/MetaMask/eth-json-rpc-infura/pull/72) is merged, I will remove this change.